src/styles/blocks/calendar.scss

```
@@ -0,0 +1,57 @@
+.calendar {
+  &__day {
+    position: relative;
+
+    @include daySize ($calendar-day-size);
+    border: 1px solid #000;
+
+    background-color: $calendar-day-color;
+    transition: 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: #ffc0cb;
+
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+@for $index from 28 through 31 {
+  .calendar--month-length_#{$index} {
+    @for $day-index from $index + 1 through 32 {
+      .calendar__day:nth-child(#{$day-index}) {
+        display: none;
+      }
+    }
+  }
+}
+
+@for $index from 1 to 32 {
+  .calendar__day:nth-child(#{$index})::after {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+
+    content: "#{$index}";
+
+    transform: translate(-50%, -50%);
+  }
+}
+
+$days:
+  "mon" 0,
+  "tue" $gap + $calendar-day-size,
+  "wed" 2 * $gap + 2 * $calendar-day-size,
```

don't repeat this calculation manually, use a number to calculate it in loop
```suggestion
  "mon" 0,
  "tue" 1,
  "wed" 2,
```